web/clusters.go

```diff
@@ -333,34 +342,60 @@ func NewClustersTable(clusters map[string]*cluster.Cluster, hostList hosts.HostL
 		}
 	}
 
-	return clusterTable
+	sort.Slice(clusterTable, func(i, j int) bool {
```

Comment:
Is this piece of code fixing the mentioned bug?

Reply:
Yes, it sorts the slice so the results are deterministic